src/main/java/pl/mimuw/zpp/quantumai/backendui/model/SolutionFile.java

```diff
@@ -0,0 +1,14 @@
+package pl.mimuw.zpp.quantumai.backendui.model;
+
+import lombok.Builder;
+import lombok.With;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Builder
+@Document
+@With
+public record SolutionFile(
+        @Id String id,
+        byte[] data
```

Comment:
z tego co widzę w internecie, ludzie korzystają z `org.bson.types.Binary`, więc może też zróbmy to w ten sposób?